great-idea/css/index.css

```diff
@@ -116,25 +124,30 @@ nav a {
 }
 
 .main-content {
-	/*margin-top: 30px;*/
-	/*margin-bottom: 30px;*/
-	padding-bottom: 30px;
+	display: flex;
+	flex-direction: column;
+	padding: 30px;
 	border-bottom: 2px solid;
 }
 
-.top-content .text-content:first-child {
-	padding-right: 10px; ;
+.top-content{
+	justify-content: space-between;
+	flex-direction: row;
+	display: flex;
 }
 
-.bottom-content .text-content:first-child, .bottom-content .text-content:nth-child(2n) {
-	padding-right: 10px; ;
+.bottom-content {
+	justify-content: space-between;
+	flex-direction: row;
+	display: flex;
 }
 
+
 .text-content {
-	display: table-cell;
-	width: 30%;
-	justify-content: space-between;
-	padding: 30px 0 30px 0;
+	/*display: table-cell;*/
+	width: 60%;
+	/*justify-content: space-between;*/
+	padding: 20px 0 20px 0;
```

Comment:
Because there's 0 spacing on the left and right of the text-content elements, some of the text in those sections is butting up against the other blocks of text. I'd recommend adding a slight padding or margin to the sides of these elements to avoid this.